sql3/test/defs/defs.go

```diff
@@ -202,3 +209,27 @@ func timestampFromString(s string) time.Time {
 	}
 	return tm
 }
+
+func operatorPresentAtPath(jplan []byte, path string, operator string) error {
```

Comment:
I feel like this function really wants a top level description of why it takes these inputs and what it does with them. I can sort of puzzle it out *what* it does from reading it, but it's much less obvious to me *why* that is a desireable thing to do.

Reply:
Yep - done.